src/js/components/Button/__tests__/Button-test.js

```diff
@@ -3,39 +3,39 @@ import React from 'react';
 import 'jest-styled-components';
 import 'jest-axe/extend-expect';
 import 'regenerator-runtime/runtime';
+import '@testing-library/jest-dom';
 
 import { axe } from 'jest-axe';
 import { Add, Next } from 'grommet-icons';
-import { cleanup, fireEvent, render } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import { Grommet, Button, Text } from '../..';
 
 describe('Button', () => {
-  afterEach(cleanup);
-
   test('should have no accessibility violations', async () => {
-    const { container, getByText } = render(
+    const { container } = render(
       <Grommet>
         <Button a11yTitle="Test button" label="Test" onClick={() => {}} />
       </Grommet>,
     );
 
-    fireEvent.click(getByText('Test'));
+    fireEvent.click(screen.getByText('Test'));
```

Comment:
Can you please add a few words of why you think your enhancement is better than the previous approach?

Reply:
In this case, I don't think we'll have a difference.

Importing `screen` directly from the `'@testing-library/react'` will only make it easier since we won't have to get `getByText`, `getByRole`, etc, from the `render` function. And it follows the official docs example: https://testing-library.com/docs/react-testing-library/example-intro#full-example